CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ include(cmake/Dependencies.cmake)
 set(AMDGPU_TARGETS gfx803;gfx900;gfx906;gfx908 CACHE STRING "List of specific machine types for library to target")
 
 # Setup VERSION
-rocm_setup_version(VERSION "2.9.0")
+rocm_setup_version(VERSION "2.10.1")
```

Comment:
Do not increment minor version. Leave at 2.9.0. I will increment version.

Reply:
Changed the version back to 2.9.0